lib/Doctrine/DBAL/Migrations/Version.php

```diff
@@ -1,93 +1,81 @@
 <?php
 
+declare(strict_types=1);
+
 namespace Doctrine\DBAL\Migrations;
 
+use Doctrine\DBAL\Connection;
 use Doctrine\DBAL\Migrations\Configuration\Configuration;
 use Doctrine\DBAL\Migrations\Event\MigrationsVersionEventArgs;
 use Doctrine\DBAL\Migrations\Provider\LazySchemaDiffProvider;
 use Doctrine\DBAL\Migrations\Provider\SchemaDiffProvider;
 use Doctrine\DBAL\Migrations\Provider\SchemaDiffProviderInterface;
 use Doctrine\DBAL\Types\Type;
+use function array_map;
+use function count;
+use function implode;
+use function is_array;
+use function is_bool;
+use function is_int;
+use function is_string;
+use function microtime;
+use function round;
+use function rtrim;
+use function sprintf;
+use function ucfirst;
 
-/**
- * Class which wraps a migration version and allows execution of the
- * individual migration version up or down method.
- *
- * @license     http://www.opensource.org/licenses/lgpl-license.php LGPL
- * @link        www.doctrine-project.org
- * @since       2.0
- * @author      Jonathan H. Wage <jonwage@gmail.com>
- */
 class Version
 {
-    const STATE_NONE = 0;
-    const STATE_PRE  = 1;
-    const STATE_EXEC = 2;
-    const STATE_POST = 3;
+    public const STATE_NONE = 0;
```

Comment:
We should extract these to standalone constants-only class(es).

Reply:
Added issue for this here #628